commons/types/Availability.d.ts

```diff
@@ -5,16 +5,21 @@ declare namespace Availability {
     slot_size: 15 | 30 | 60;
     start_date: Date;
     dates_to_show: number;
-    available_times: TimeSlot[];
     show_ticks: boolean;
+    calendars: Calendar[];
   }
 
+  interface Calendar {
+    email_address: string;
```

Comment:
```suggestion
    emailAddress: string;
```

Reply:
Why `emailAddress` ?

Reply:
Trying to match up with my comment below about only having external props in `snake_case`